packages/react-ui-core/src/Menu/Menu.js

```diff
@@ -93,28 +95,26 @@ export default class Menu extends PureComponent {
     }
   }
 
-  get value() {
-    return this.options[this.state.highlightIndex]
-  }
-
   get options() {
     return this.props.options || []
   }
 
   get items() {
-    return this.options.map(option => (
-      typeof option === 'object' ? option.label : option
-    ))
+    return this.options.map(option => ((typeof option === 'object' && option.label !== undefined) ? option.label : option))
+  }
+
+  option() {
```

Comment:
What do you think about renaming this to `highlightedOption` to make it more clear? Also I see no reason this shouldn't be a getter.

Reply:
Fixed